pinc/ProofreadingToolbox.inc

```diff
@@ -1,375 +1,111 @@
-            "top.surroundSelection('[Illustration: ',']')"
-        );
-        $this->define_tool_link(
-            'note',
-            '[** ]',
-            _('Note'),
-            "top.surroundSelection('[** ',']')"
-        );
-        $this->define_tool_link(
-            'brackets',
-            '[ ]',
-            '[ ]',
-            "top.surroundSelection('[',']')"
-        );
-        $this->define_tool_link(
-            'braces',
-            '{ }',
-            '{ }',
-            "top.surroundSelection('{','}')"
-        );
-        $this->define_tool_link(
-            'footnote',
-            '[Footnote #: ]',
-            '[Footnote #:',
-            "top.surroundSelection('[Footnote #: ',']')"
-        );
-        $this->define_tool_link(
-            'slash_star',
-            '/* */',
-            _('Start poetry/table'),
-            "top.surroundSelection('/*\\n','\\n*/')"
-        );
-        $this->define_tool_link(
-            'slash_number',
-            '/# #/',
-            _('Start block quote'),
-            "top.surroundSelection('/#\\n','\\n#/')"
-        );
-        $this->define_tool_link(
-            'thought_break',
-            '<tb>',
-            _('Thought Break'),
-            "top.surroundSelection('\\n<tb>\\n','')"
-        );
-        $this->define_tool_link(
-            'blank_page',
-            '[Blank Page]',
-            '[Blank Page]',
-            "top.replaceAllText('[Blank Page]')"
-        );
-    }
+        global $code_url;
+        $formatting = is_formatting_round($round);
 
-    private function array_extract( $source, $keys )
-    {
-        if ( $keys == 'ALL' )
+        echo "<div id='tools'>";
+        foreach ($this->tool_buttons as $tool_button)
         {
-            return $source;
+            $tool_button->draw($formatting);
         }
 
-        $result = array();
-        foreach ( $keys as $key )
+        draw_popup_link(_('Search/Replace'), 'srchrep.php', 'dp_searchrepl', 300, 250);
+        if ($formatting)
         {
-            if ( array_key_exists($key, $source) )
-            {
-                $result[$key] = $source[$key];
-            }
-            else
-            {
-                // It's tempting to do an assert(0), but $key could be for a tool
-                // that *can* exist, but happens not to in this installation.
-                // (i.e., currently, hieroglyph_transliterator)
-            }
+            spacer_bar();
+            draw_popup_link(_('Table Maker'), 'mktable.php', 'dp_mktable', 600, 500);
         }
-        return $result;
+        spacer_bar();
+        draw_popup_link(_('Greek Transliterator'), 'greek2ascii.php', 'gkasciiWin', 640, 250);
+
+        $url = get_faq_url($round->document);
+        echo "[<a href='$url' class='nowrap' target='roundDoc'>", _('Guidelines'), "</a>]\n";
+
+        echo "[<a href='$code_url/faq/InterfaceDiagram.png' class='nowrap' target='ifaceWin'>", _("Proofreading Diagram"), "</a>]\n";
```

Comment:
I think we should remove this diagram link altogether, but will defer to @srjfoo and @lhamilton1.

Reply:
Okay by me. It's not up-to-date, anyway (still says that reporting bad pages can be done only from the first round), and doesn't mention format preview.

Reply:
Link removed.